src/pages/user.tsx

```diff
@@ -12,6 +39,9 @@ export function User() {
     <div className="flex flex-col h-full items-center">
       <div className="mt-9 pb-3 gap-8 align-middle items-center grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-6">
         <AddButton onClick={openModal} />
+        {userData.map((user, index) => (
+          <UserCard data={user} key={index} />
```

Comment:
Suggestion: use another thing to pass to the key property. Usually, React performs badly with number keys related to the index, it is better to have something that is unique to the element, like ids, names, emails... In this case, user.id would do the work.

Reply:
yeah sure i changed to key={user.id}